src/Css/Stylesheet.php

```diff
@@ -1612,6 +1612,7 @@ private function _parse_font_face($str)
      */
     private function _parse_properties($str)
     {
+        $properties = preg_split("/;(?=(?:[^\(]*\(.*?[^\)]*\))*(?![^\)]*\)))/", $str);
```

Comment:
Just a quick note: It seems this line was added during merge, due to the changes on master. It is obsolete now.

Reply:
ah thank you I thought I had removed that